src/main/java/com/google/api/codegen/transformer/py/PythonSurfaceNamer.java

```diff
@@ -453,4 +453,27 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
         throw new IllegalStateException("Invalid development status");
     }
   }
+
+  @Override
+  public String getPrintSpec(String spec) {
```

Comment:
I'm finding that `\n` in a quoted string in the YAML input does not show up in the output. I have to double-escape in the YAML `\\n` to get it to show up as `\n` in the output. It would be great if we don't have to double-escape in the YAML. Do you know how we can achieve that?

Reply:
Oh, that was with double-quoted strings. Single-quoted strings appear to pass escape sequences without double escaping. Sweet.